src/libs/BankAccountUtils.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'underscore';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+import DateUtils from './DateUtils';
+import * as Localize from './Localize';
+
+/**
+ * Helper method to build the Onyx data required during setup of a Verified Business Bank Account
+ *
+ * @returns {Object}
+ */
+// We'll remove the below once this function is used by the VBBA commands that are yet to be implemented
+function getVBBADataForOnyx() {
+    return {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                value: {
+                    isLoading: true,
+                    errors: null,
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                value: {
+                    isLoading: false,
+                    errors: null,
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                value: {
+                    isLoading: false,
+                    errors: {
+                        [DateUtils.getMicroseconds()]: Localize.translateLocal('paymentsPage.addBankAccountFailure'),
+                    },
+                },
+            },
+        ],
+    };
+}
+
+/**
+ * Helper method to get the onfido sdk token from the bank account's achdata
+ *
+ * @param {Object} achData
+ * @returns {string}
+ */
+function getOnfidoSDKTokenFromACHData(achData) {
+    const sdkToken = _.get(achData, ['verifications', 'externalApiResponses', 'requestorIdentityOnfido', 'apiResult', 'sdkToken'], '');
+    return sdkToken || _.get(achData, ['sdkToken'], '');
```

Comment:
Let's standardize on whether we want to get rid of the `achData` key or keep it. https://github.com/Expensify/Web-Expensify/pull/34821#discussion_r973975792

Reply:
Looks like we're keeping it yeah?

Reply:
That doesn't affect this PR though it seems. Right? since it's already expecting verifications to be in `achData` which it seems is where it still is after that update?

Reply:
Before the update the verifications would be in achData > additionalData > verifications.

You must've seen verifications under achData because the previous steps were calling the deprecated command which returned the correct format. So I think you should be good here.